pkg/kubelet/prober/worker.go

```diff
@@ -152,7 +152,7 @@ func (w *worker) doProbe() (keepGoing bool) {
 	}
 
 	c, ok := api.GetContainerStatus(status.ContainerStatuses, w.container.Name)
-	if !ok {
+	if !ok || len(c.ContainerID) == 0 {
 		// Either the container has not been created yet, or it was deleted.
 		glog.V(3).Infof("Non-existant container probed: %v - %v",
```

Comment:
Although the log below says "Non-running container probed" (presumably meaning it used to be running and has since terminated). How about if I just change the spelling to Non-existent?